inc/ThirdParty/Themes/Divi.php

```diff
@@ -198,4 +200,18 @@ public function exclude_css_from_combine( $exclude_css ) {
 
 		return $exclude_css;
 	}
+
+	/**
+	 * Disable Divi dynamic CSS when RUCSS is activated
+	 *
+	 * @return void
+	 */
+	public function disable_dynamic_css_on_rucss() {
+
+		if ( ! apply_filters( 'pre_get_rocket_option_remove_unused_css', false ) ) { // phpcs:ignore WordPress.NamingConventions.PrefixAllGlobals.NonPrefixedHooknameFound
```

Comment:
Is there a reason to use the filter instead of using the option value from the injected `$options` instance?

Reply:
@Tabrisrp no I just didn't knew we can get it this way